### DISCUSSION TITLE
How to setup SourceLink with NuGet and CloudSmith packages?

### DISCUSSION THREAD

**Discussion by xperiandri** [OrchardCMS/OrchardCore#10092] [Q&A]
Is it available for NuGet packages?
How to add CloudSmith server as SourceLink?

**Comment by ns8482e** [comment]
Related Issue #9990  - For nuget.org  `snupkgs` were excluded when published 1.0 
Probably @sebastienros can manually publish or we have to wait for next version release.

**Reply by xperiandri** [reply]
What about CloudSmith? We currently use preview builds
